Mathlib/Algebra/Module/LinearMap/Basic.lean

```diff
@@ -831,6 +832,32 @@ instance [DistribMulAction Sᵐᵒᵖ M₂] [SMulCommClass R₂ Sᵐᵒᵖ M₂]
     IsCentralScalar S (M →ₛₗ[σ₁₂] M₂) where
   op_smul_eq_smul _ _ := ext fun _ ↦ op_smul_eq_smul _ _
 
+variable {S' T' : Type*}
+variable [Monoid S'] [DistribMulAction S' M] [SMulCommClass R S' M]
+variable [Monoid T'] [DistribMulAction T' M] [SMulCommClass R T' M]
+
+instance instDomSmul : SMul S'ᵈᵐᵃ (M →ₛₗ[σ₁₂] M₂) where
+  smul a f :=
+    { toFun := a • (f : M → M₂)
+      map_add' := fun x y ↦ by simp only [DomMulAct.smul_apply, f.map_add, smul_add]
+      map_smul' := fun c x ↦ by simp_rw [DomMulAct.smul_apply, ← smul_comm, f.map_smulₛₗ] }
+
+theorem _root_.DomMulAct.smul_linearMap_apply (a : S'ᵈᵐᵃ) (f : M →ₛₗ[σ₁₂] M₂) (x : M) :
+    (a • f) x = f (DomMulAct.mk.symm a • x) :=
+  rfl
+
+@[simp]
+theorem _root_.DomMulAct.mk_smul_linearMap_apply (a : S') (f : M →ₛₗ[σ₁₂] M₂) (x : M) :
+    (DomMulAct.mk a • f) x = f (a • x) :=
+  rfl
+
+theorem  _root_.DomMulAct.coe_smul_linearMap (a : S'ᵈᵐᵃ) (f : M →ₛₗ[σ₁₂] M₂) :
+    (a • f : M →ₛₗ[σ₁₂] M₂) = a • (f : M → M₂) :=
+  rfl
```

Comment:
I renamed these lemmas to match the 6 or so that we already have. If you're not a fan of them, then let's rename all of them at once.

Reply:
Thanks, I like your version better